app/templates/serviceLearning/slcProposal.html

```diff
@@ -24,8 +24,8 @@ <h2 class="text-center pt-4 pb-2"> New Proposal For Service-Learning Course</h2>
             <tr data-username="" hidden>
                 <td>
                   <p class="mb-0"></p>
-                    <input type="text" style="border: none" size="12" class="form-control-sm" id="" name="courseInstructorPhone" aria-label="Instructor Phone" value="" placeholder="Phone Number" />
-                    <a class="text-decoration-none primary editButton" tabindex="0" data-id=""  id=""  type="button">Edit</a>
+                    <input type="text" style="border: none" size="14" class="form-control-sm" id="" name="courseInstructorPhone" aria-label="Instructor Phone" data-value="" value="" placeholder="Phone Number" />
```

Comment:
Why is the size of the input 14? Is it due to the phone number's formatting?

Reply:
Yes, if it is 12 there are some weird cases on some devices that cut the phone number display off by one or two numbers.